src/appconfig.rs

```diff
@@ -36,3 +44,100 @@ pub async fn connect_to_database_and_migrate(database_url: &str) -> PgPool {
         .expect("Failed to migrate database.");
     connection_pool
 }
+
+fn get_cors(settings: Settings) -> Cors {
+    let mut cors = Cors::default();
+    for origin in allowed_origins(&settings) {
+        cors = cors.allowed_origin(origin);
+    }
+    cors = cors.allow_any_method();
+    cors = cors.allowed_headers(vec![http::header::ACCEPT, http::header::CONTENT_TYPE]);
+    cors
```

Comment:
This doesn't need to be `mut`, or assigned. The example here shows the typical chaining build version:
https://docs.rs/actix-cors/latest/actix_cors/#example
Which can just be returned instead of building the `mut`.

Reply:
Hmm...I did this because I couldn't see a method for adding multiple origins at once so I needed the for loop. How do I achieve this in a chain?

Reply:
Your comment changed from what I got in email so I got confused.

Reply:
The last two lines could be tightened up by adding them onto the definition line after `default()`.

Reply:
thank you